uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request.controller.spec.js

```diff
@@ -593,6 +593,31 @@
       });
 
       describe('manager opens leave request popup in edit mode', function () {
+        describe('when leave request date has same date as period end date', function () {
```

Comment:
`when leave request "to" date is same as absence period end date`

We also miss a case

`when leave request "from" date is same as absence period start date`